internal/impl/mongodb/input.go

```diff
@@ -42,23 +44,39 @@ func mongoConfigSpec() *service.ConfigSpec {
 		Field(service.NewBloblangField("query").
 			Description("Bloblang expression describing MongoDB query.").
 			Example(`
-  root.from = {"$lte": timestamp_unix()}
-  root.to = {"$gte": timestamp_unix()}
-`))
+root.from = {"$lte": timestamp_unix()}
+root.to = {"$gte": timestamp_unix()}
+`)).
+		Field(service.NewAnyMapField("options").
```

Comment:
I'd prefer to have explicit fields rather than a generic option as it makes documentation and linting more explicit, is there a specific reason for using an AnyMap here?

Reply:
Fine by me.  Changed.